### ISSUE TITLE
v0.10.1 bails out on ' in description element

### ISSUE THREAD

**Issue by jdall** [asyncapi/generator#102]
Running 0.9.0, our A2S document was fine, but with 0.10.1 the generator just spews out:
```
Something went wrong:
Error: Invalid AsyncAPI document
    at parse (/usr/lib/node_modules/asyncapi-generator/node_modules/asyncapi-parser/lib/parser.js:93:11)
    at <anonymous>
```
After some debugging, it was clear that the root cause was a ' in a description element, as in the following example:
```
asyncapi: '2.0.0-rc1'
id: 'urn:hello-world-app'
info:
  title: Hello world application
  version: '0.1.0'
channels:
  hello:
    description: "subscribe to hello's"
    subscribe:
      message:
        payload:
          type: string
          pattern: '^hello .+$'
```
It's probably us not following the A2S spec (though I haven't been able to find anything that says it's *not* allowed), but could the generator be a bit more helpful as to why it failed?

Thanks

**Comment by fmvilas** [comment]
That is definitely a bug. Single quotes must be allowed. Thanks for reporting!

**Comment by jdall** [comment]
Fixed in 0.13.1
